e2-studio-star-rx72n-firmware/libs/rx_hcsr04/src/rx_hcsr04_icu.c

```
@@ -0,0 +1,127 @@
+ * Static Variables
+ * =============================================================================
+ */
+
+static const char* s_tag = "HCSR04_ICU"; /**< Logging tag */
+
+/* =============================================================================
+ * Public Functions
+ * =============================================================================
+ */
+
+rx_err_t rx_hcsr04_icu_configure(const uint8_t irq_num, const uint8_t priority)
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'rx_hcsr04_icu_configure' of similar type ('const uint8_t') are easily swapped by mistake

(bugprone-easily-swappable-parameters)